src/swap/SwapPending.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { useEffect, useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ActivityIndicator, BackHandler, StyleSheet, Text, View } from 'react-native'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { useSelector } from 'react-redux'
+import Button, { BtnTypes } from 'src/components/Button'
+import Dialog from 'src/components/Dialog'
+import Checkmark from 'src/icons/Checkmark'
+import Times from 'src/icons/Times'
+import { noHeader } from 'src/navigator/Headers'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { swapStateSelector } from 'src/swap/selectors'
+import { SwapState } from 'src/swap/slice'
+
+export function SwapPending() {
```

Comment:
`SwapPendingScreen`? `SwapExecuteScreen`?

Reply:
Changed in ede3eced2714a40c7a6c9796bbef588523a06739!